Marlin/src/pins/esp32/pins_MKS_TINYBEE.h

```diff
@@ -170,11 +170,9 @@
   #define BTN_EN2                    EXP2_05_PIN
 
   #if ENABLED(FYSETC_MINI_12864_2_1)
-    // MKS_MINI_12864_V3, BTT_MINI_12864_V1, FYSETC_MINI_12864_2_1
-    #define DOGLCD_CS                EXP1_03_PIN
-    #define DOGLCD_A0                EXP1_04_PIN
-    #define LCD_RESET_PIN            EXP1_05_PIN
-    #define NEOPIXEL_PIN             EXP1_06_PIN
+
+  // Migrated to pins/lcd
+
     #if SD_CONNECTION_IS(ONBOARD)
       #define FORCE_SOFT_SPI
```

Comment:
Keep an eye on the defines of `FORCE_SOFT_SPI`. I think we can probably automate the detection of cases where this flag is necessary. Specifically, if the SD SPI interface is defined to use pins that are not on the SPI bus, then this flag should be set. Maybe there are cases where it should be set even when the pins are on the SPI bus, but I would find that strange.